packages/hint-compat-api/src/helpers/compat-html.ts

```diff
@@ -8,6 +8,8 @@ import { MDNTreeFilteredByBrowsers, TestFeatureFunction, FeatureInfo } from '../
 import { CompatStatement } from '../types-mdn.temp';
 import { CompatBase } from './compat-base';
 
+export const DEFAULT_HTML_IGNORE = ['link-integrity'];
```

Comment:
Two things:
1. The `integrity` attribute is also defined for `<script>` so that needs to be covered as well.
2. Should we implement this such that we can just specify `integrity` in the ignore list?

I ask the second because I don't think we should expect users to know how we map this to a `featureName` internally.

Reply:
Sorry, my fault. It should be integrity 👍. Thanks to this scenario I found I was missing to exclude sub features.

Reply:
Let's include a test case with both `<link>` and `<script>` elements with `integrity` as well just to validate this is working as expected.

Reply:
Sure!